manual/src/getting_started/run_examples.md

```diff
@@ -0,0 +1,45 @@
+# Run examples
+
+## Native on desktop
+
+You can find examples in the `examples/` directory.
+
+You can start the widgets example by executing the following command:
+
+```text
+cargo run --example widgets --release
+```
+
+## Run Examples with cargo-node
+
+To run the examples on as browser, electron or cordova app you have to install
+
+```text
+cargo install -f cargo-node
+```
+
+Before you could use cargo node you have to install `npm` version 6.9.0. It is included in the `Node.js` version 10.16.3. You could download it from https://nodejs.org/dist/v10.16.3/. 
```

Comment:
"could" => "can" (it currently reads as if you can't do this anymore). Same problem with the root readme.

Reply:
Ok thank you.

Reply:
"on as browser" --> "in a browser"